### PR TITLE
[zh-cn]: create the translation of `navigator.presentation` property

### PR DESCRIPTION
**Pull request by T34-active**

### CODE REVIEW

File: files/zh-cn/web/api/navigator/presentation/index.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+title: Navigator：presentation 属性
+slug: Web/API/Navigator/presentation
+l10n:
+  sourceCommit: b0c8b07682c8d2cecc544f60468f3cf6fc20ac99
+---
+
+{{securecontext_header}}{{APIRef("Presentation API")}}
+
+{{DOMxRef("Navigator")}} 接口的 `presentation` 只读属性是 [演示 API](/zh-CN/docs/Web/API/Presentation_API) 的入口点，用于返回对 {{DOMxRef("Presentation")}} 对象的引用。
```
Comment (yin1999) [inline files/zh-cn/web/api/navigator/presentation/index.md:10]: https://www.w3.org/zh-hans/news/2016/w3c-invites-implementations-of-presentation-api/

```suggestion
{{DOMxRef("Navigator")}} 接口的 `presentation` 只读属性是[呈现 API](/zh-CN/docs/Web/API/Presentation_API) 的入口点，用于返回对 {{DOMxRef("Presentation")}} 对象的引用。
```

File: files/zh-cn/web/api/navigator/presentation/index.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+## 语法
+
+```js-nolint
+const presentation = navigator.presentation
+```
+
+### 值
+
+参考 {{DOMxRef("Presentation")}} 对象。
```
Comment (yin1999) [inline files/zh-cn/web/api/navigator/presentation/index.md:20]:
```suggestion
{{DOMxRef("Presentation")}} 对象的引用。
```

File: files/zh-cn/web/api/navigator/presentation/index.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+{{Specifications}}
+
+## 浏览器兼容性
+
+{{Compat}}
+
+## 参见
+
+- [演示 API](/zh-CN/docs/Web/API/Presentation_API)
```
Comment (yin1999) [inline files/zh-cn/web/api/navigator/presentation/index.md:32]:
```suggestion
- [呈现 API](/zh-CN/docs/Web/API/Presentation_API)
```

**Review by yin1999 — CHANGES_REQUESTED** [review]

**Review by yin1999 — APPROVED** [review]
:+1:

**Comment by T34-active** [comment]
@yin1999 Thank you!

**Pull request merged**